.github/workflows/release.yml

```diff
@@ -0,0 +1,65 @@
+name: Create release
+on: [push]
+
+jobs:
```

Comment:
Perhaps could be useful to add pypi publishing too (might be out of scope for this pr)

Reply:
https://github.com/yt-dlp/yt-dlp-sample-plugins/issues/5